packages/odata-service-writer/src/data/manifest.ts

```diff
@@ -275,7 +260,7 @@ function addRemoteAnnotationDataSources(
                     )}',Version='0001')/$value/`,
                     type: 'ODataAnnotation',
                     settings: {
-                        localUri: resolveLocalServiceUri(serviceName, remoteAnnotation.technicalName)
+                        localUri: `localService/${serviceName}/${trimSlashes(remoteAnnotation.technicalName)}.xml`
```

Comment:
Why not fix the existing code rather than adding only a selective change. Dont we need to sanitize the service name also?